### PR TITLE
change the name of default snapshot

### LINKED ISSUE #125: Change the default snapshot name from "snapshot" to something else.
This snapshot already exists causes a lot of confusion to lot of users. If ceph is completely untouched it should still work, but sadly that may not be the case always. So we should probably have the default name changed to some UUID,

### PR DESCRIPTION
**Pull request by naved001**
Fixes #125

### CODE REVIEW

File: ims/common/constants.py [hunk 1]
```diff
@@ -120,7 +120,7 @@
 DNSMASQ_LEASES_LOC = '/var/lib/misc/dnsmasq.leases'
 
 HIL_CALL_TIMEOUT = 10
-DEFAULT_SNAPSHOT_NAME = "snapshot"
+DEFAULT_SNAPSHOT_NAME = "bmi-created-snapshot"
```
Comment (apoorvemohan) [inline ims/common/constants.py]: I think this should be the UUID specified in the bmiconfig.cfg file instead of hardcoding it. What say?

Comment (naved001) [reply ims/common/constants.py]: does it have to be a UUID, or just any string in the config file.

Comment (apoorvemohan) [reply ims/common/constants.py]: i think either would work for now

**Comment by naved001** [comment]
Made the change you suggested, but one of the tests is now reading from a configuration file that I don't know about. *sigh*. And that's why it's failing the tests.

**Review by ianballou — COMMENTED** [review]
This looks fine to me, I'll +1 when tests pass.

**Comment by apoorvemohan** [comment]
LGTM

**Review by apoorvemohan — APPROVED** [review]

**Comment by naved001** [comment]
now I need to dig deep in the CI.

**Review by ianballou — APPROVED** [review]

**Comment by naved001** [comment]
closing this in favor of #187

**Pull request closed without merging**